docs/hazmat/backends/interfaces.rst

```diff
@@ -422,14 +422,15 @@ A specific ``backend`` may provide one or more of these interfaces.
         :returns: An instance of
             :class:`~cryptography.hazmat.primitives.asymmetric.ec.EllipticCurvePublicKey`.
 
-    .. method:: derive_elliptic_curve_public_point(private_value, curve)
+    .. method:: derive_elliptic_curve_public_key(private_value, curve)
```

Comment:
I realize the value you're computing is the public  key, but since you have the private value, is there a reason not to return a private key instance?